src/modules/information/types.ts

```diff
@@ -0,0 +1,17 @@
+import { ScopePublicity } from '@prisma/client';
+
+export type UpdateInformationProps = {
+  bio?: string;
+  avatarUrl?: string;
+  coverImageUrl?: string;
+  emailPublicity?: ScopePublicity;
+  phone?: string;
+  phonePublicity?: ScopePublicity;
+  facebookUrl?: string;
+  facebookPublicity?: ScopePublicity;
+  dateOfBirth?: Date | string;
+  dateOfBirthPublicity?: ScopePublicity;
+  gradeCode?: string;
+  gradeName?: string;
+  className?: string;
```

Comment:
we do not allow edit grade and class

Reply:
copied from prisma client :v

Reply:
please add some logic to prevent users from changing class